### ISSUE TITLE
Containers started by Openshift have no logs

### ISSUE THREAD

**Issue by deanpeterson** [openshift/origin#4790]
I can take the exact same image created by Openshift and manually run it outside openshift with the same CMD and I can see logs using the "docker logs..." command.  However, the container created by Openshift and Kubernetes does not print anything with the "docker logs..." command.

For example, when I run docker ps -a, I can see: 
a77de0586930        172.30.239.80:5000/test/tradeservice@sha256:29246b264315d43632bdd119cca7a283b7c7b716f1b3b1de284a41e177772223   "/bin/sh -c '/opt/jb   About an hour ago                                                        k8s_wildfly-tradeservice.79fba1ed_tradeservice-2-s8iu1_test_0d7703c7-62e6-11e5-938b-0023ae6f50aa_0dfe605f   

When I run docker logs k8s_wildfly-tradeservice.79fba1ed_tradeservice-2-s8iu1_test_0d7703c7-62e6-11e5-938b-0023ae6f50aa_0dfe605f 
or docker logs a77de0586930, absolutely nothing is returned.  It just moves to a new line on the terminal.

When I manually run a container with the same image:     docker run 172.30.239.80:5000/test/tradeservice@sha256:29246b264315d43632bdd119cca7a283b7c7b716f1b3b1de284a41e177772223 /bin/sh -c '/opt/jboss/wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement=0.0.0.0 --properties=/opt/jboss/wildfly/tradeapp.properties'      

Then, I call docker logs  <container name> for that container; I see the wildfly server starting as I would expect.  Somehow, the container being started by Kubernetes is different even though it looks like it is calling /bin/sh -c '/opt/jboss/wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement=0.0.0.0 --properties=/opt/jboss/wildfly/tradeapp.properties' by looking at what is printed out on docker ps -a. 

<b>There is an associated error in the event logs: </b>

failed
Failed to start with docker id 99a811606b51 with error: API error (500): Cannot start container 99a811606b51cdbeddbea14af474f0df432278ac9f73baea5d8ecaf5453f521e: cannot join network of a non running container: c8a4a648e63d5b18dcd8fad6fd2d70f466584f2a749bb4245bb86a6da1ceea55

I am only trying to start one container, why is it complaining about not being able to start one container because another is not running?

**Comment by deanpeterson** [comment]
There is an associated error in the event logs: 

failed
Failed to start with docker id 99a811606b51 with error: API error (500): Cannot start container 99a811606b51cdbeddbea14af474f0df432278ac9f73baea5d8ecaf5453f521e: cannot join network of a non running container: c8a4a648e63d5b18dcd8fad6fd2d70f466584f2a749bb4245bb86a6da1ceea55

I am only trying to start one container, why is it complaining about not being able to start one container because another is not running?

**Comment by deanpeterson** [comment]
I opened up a bugzilla bug; this is a showstopper for me - https://bugzilla.redhat.com/show_bug.cgi?id=1266533

**Comment by deanpeterson** [comment]
I figured out the problem.  I vaguely remembered seeing this in the documentation and didn't think much of it at the time: https://docs.openshift.com/enterprise/3.0/admin_guide/manage_scc.html#enable-images-to-run-with-user-in-the-dockerfile

I have the line "USER root" in my Dockerfile.  Apparently that was preventing the container from starting.  After I followed the instructions to modify my scc to RunAsAny, the deployment was successful.

Granted, I should have read the documentation better, but an error message telling me my containers do not have the privilege to run would have helped immensely.
